package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "the-events-calendar",
-  "version": "6.8.3",
+  "version": "7.0.0",
```

Comment:
major version update for TEC as well ? Something seems off here

Reply:
Nope - test version number.